frontend/containers/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+import {Component, h} from 'preact'
+import proptypes from 'proptypes'
+import {connect} from 'preact-redux'
+import {bindActionCreators} from 'redux'
+import {route} from 'preact-router'
+
+import Style from 'lib/Style'
+import styles from './UserProfile.css'
+
+import {Case, connectHelper, setPageTitle} from 'lib/util'
+import {getAuthCollection} from 'lib/app-config'
+import {buildUrl} from 'lib/router'
+
+import * as userActions from 'actions/userActions'
+
+import SubNavItem from 'components/SubNavItem/SubNavItem'
+
+/**
+ * The interface for editing a user profile.
+ */
+class UserProfile extends Component {
```

Comment:
Seems to be on the wrong line so this isn't being captured by updates, however it has been resolved